pysoa/test/stub_service.py

```diff
@@ -372,20 +372,20 @@ def __init__(self, service, action, body=None, errors=None, side_effect=None):
         self._stub_action_responses_to_merge = defaultdict(dict)
 
     def __enter__(self):
-        mock_action = self._MockAction(name='{}.{}'.format(self.service, self.action))
-
         if self.enabled:
-            return mock_action
+            return self.mock_action
+
+        self.mock_action = self._MockAction(name='{}.{}'.format(self.service, self.action))
```

Comment:
Can we call this `self._current_mock_action`, instead, please? Also, we should initialize `self._current_mock_action = None` in the constructor.

Reply:
Fixed, thanks!